src/main/java/org/folio/fqm/service/QueryManagementService.java

```diff
@@ -223,8 +219,7 @@ private static Date offsetDateTimeAsDate(OffsetDateTime offsetDateTime) {
   }
 
   private List<String> getFieldsFromEntityType(EntityType entityType) {
-    return entityType
-      .getColumns()
+    return (entityType.getColumns() != null ? entityType.getColumns() : Collections.<EntityTypeColumn>emptyList())
```

Comment:
This is just a temp stub to deal with nullable columns so that tests pass. We just need to add a default value for `columns` in folio-query-tool-metadata, and we can then remove this change